upcoming/composable-pipelines-pattern.md

```diff
@@ -0,0 +1,392 @@
++++
+Author = ["Samuel Lampa"]
+date = "2015-01-30T00:00:00-08:00"
+series = [""]
+title = "Patterns for composable concurrent pipelines in Go"
++++
+
+# Patterns for composable concurrent pipelines in Go
+
+## Generator function in Python
+
+The author of this post came into Go from the python-world, which is quite prevalent the field of bioinformatics. In python one can easily write composable lazy-evauated pipelines of string processing operations using the generator syntax built into the language, which has been shown to both use less memory and to be a bit faster than their eager counterparts.
+
+The Generator functionality in python basically means that you create a function that, rather than returning a single data structure once (say for example a list of items), rather will return a generator object which can later be iterated over by repeadetly calling its next method, or by the shorthand `for item in ...` syntax.
```

Comment:
Fixed (to be pushed in a minute).

You mean that `next()` is a better formatting? (fixed!)